sdks/python/apache_beam/examples/wordcount_xlang.py

```diff
@@ -70,7 +70,7 @@ def run(p, input_file, output_file):
             | 'split' >> (beam.ParDo(WordExtractingDoFn())
                           .with_output_types(bytes))
             | 'count' >> beam.ExternalTransform(
-                'pytest:beam:transforms:count', None, EXPANSION_SERVICE_ADDR))
+                'beam:transforms:xlang:count', None, EXPANSION_SERVICE_ADDR))
```

Comment:
Please create a JIRA to create a standard for cross-language transform URNs.
I think having a recommended standard will ensure (hopefully) that users do not end up creating cross-language transforms with conflicting URNs.

cc: @robertwb @mxm

Reply:
I've created Jira issue: https://issues.apache.org/jira/browse/BEAM-7904